include/envoy/network/udp_packet_writer_handler.h

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+
+#include "envoy/api/io_error.h"
+#include "envoy/buffer/buffer.h"
+#include "envoy/network/address.h"
+#include "envoy/network/socket.h"
+#include "envoy/stats/scope.h"
+#include "envoy/stats/stats_macros.h"
+
+namespace Envoy {
+namespace Network {
+
+/**
+ * Udp Default Packet Writer Name
+ **/
+constexpr absl::string_view UdpDefaultWriterName = "udp_default_writer";
```

Comment:
This is no longer needed. Just inline this. Otherwise LGTM!

/wait

Reply:
Removed UdpDefaultWriterName!